sdks/java/core/src/main/java/org/apache/beam/sdk/values/WindowedValues.java

```diff
@@ -904,11 +987,14 @@ public <NewT> WindowedValueCoder<NewT> withValueCoder(Coder<NewT> valueCoder) {
     @Override
     public void encode(WindowedValue<T> windowedElem, OutputStream outStream)
         throws CoderException, IOException {
-      encode(windowedElem, outStream, Context.NESTED);
+      encode(windowedElem, outStream, org.apache.beam.sdk.coders.Coder.Context.NESTED);
```

Comment:
`Coder.Context` would suffice here I think?
```suggestion
      encode(windowedElem, outStream, Coder.Context.NESTED);
```